### PR TITLE
Explain how to run soltest

### PR DESCRIPTION
**Pull request by axic**

### CODE REVIEW

File: docs/contributing.rst [hunk 1]
```diff
@@ -59,3 +59,15 @@ for this project. Also, even though we do CI testing, please test your code and
...
+
+Running the compiler tests
+==========================
+
+Solidity includes different types of tests. They are included in the application
+called ``soltest``. Some of them require the ``cpp-ethereum`` client in testing mode.
+
+To run ``cpp-ethereum`` in testing mode: ``eth --test -d /tmp/testeth``.
+
+To run the tests: ``soltest -- --ipcpath /tmp/testeth/geth.ipc``.
```
Comment (chriseth) [inline docs/contributing.rst:71]: If you only want to run a subset of the tests, you can use
``soltest -t TestSuite/TestName -- --ipcpath /tmp/testeth/geth.ipc`` or 
``soltest -t EndToEndTests/* -- --ipcpath /tmp/testeth/geth.ipc`` or

Comment (VoR0220) [reply docs/contributing.rst:71]: ^this needs to be added. Saves SO MUCH time.

Comment (axic) [reply docs/contributing.rst:71]: Added.

**Review by VoR0220 — APPROVED** [review]
LGTM

**Pull request merged**